packages/element-plus/package.json

```diff
@@ -17,10 +17,12 @@
   "license": "MIT",
   "main": "lib/index.js",
   "module": "es/index.mjs",
+  "types": "es/index.d.ts",
   "exports": {
     ".": {
       "require": "./lib/index.js",
-      "import": "./es/index.mjs"
+      "import": "./es/index.mjs",
+      "types": "./es/index.d.ts"
```

Comment:
This line seems to be useless...

Reply:
I think we need it after TypeScript v4.5.
refer to https://devblogs.microsoft.com/typescript/announcing-typescript-4-5-beta/#packagejson-exports-imports-and-self-referencing